modules/auxiliary/cloud/aws/enum_ec2.rb

```diff
@@ -61,15 +66,60 @@ def describe_ec2_instance(i)
     print_good "    Creation Date:  #{i.launch_time}"
     print_good "    Public IP:      #{i.public_ip_address} (#{i.public_dns_name})"
     print_good "    Private IP:     #{i.public_ip_address} (#{i.private_dns_name})"
+    # Report hosts and info
+    mac_addr = i.network_interfaces.select { |iface|
+      iface.private_ip_address == i.private_ip_address
+    }.first.mac_address
+    iname = i.tags.find {|t| t.key == 'Name'} ? i.tags.find {|t| t.key == 'Name'}.value : i.private_dns_name
+    iinfo = i.tags.find {|t| t.key == 'Description'} ? i.tags.find {|t| t.key == 'Description'}.value : nil
```

Comment:
Just a suggestion but, maybe this could be simplified:
```suggestion
    iname = i.tags['Name']&.value || i.private_dns_name
    iinfo = i.tags['Description']&.value
```

Reply:
This is a never ending debate :smile: - some people want to make it easier for devs coming into the project to read what we wrote, some want to make it more succinct. IMO, leaving the notation expanded/expressive makes it easier for Pythonists and other incoming developers to grok; however, if you feel strongly about it, i can make the change.